### ISSUE TITLE
[Feature Request]. Onscreen Feedback for Gyro Self Check

### ISSUE THREAD

**Issue by stepheneadams66** [FrSkyRC/ETHOS-Feedback-Community#4284]
Use Case.  RX is placed in a location where the Yellow LED cannot be seen.

1) Currently when you Enable Self Check and hit Enter.  The screen refreshes with the button indicating Disabled. This is misleading.  The button should say "Enabled" until after the RX manipulates the control surfaces.  

2) It would also be nice to have audible feedback to indicate it's time to move the sticks.

**Comment by Richrfl** [comment]
I agree. Not always you can see the LED to indicate when to move the sticks (you may even have to install the RX upside-down, or sideways). I guess that either a sound or some surface control flipping should indicate start moving the sticks and a full dancing sequence of all surfaces should indicate the calibration process has finished.

**Comment by frskyrc-ben** [comment]
I totally agree we need better user experience for the Self-check process.

Here are some screenshot mockups how I can see this being implemented : 

![image](https://github.com/user-attachments/assets/06bce4df-5857-4192-8bc9-48225ae2122d)

If there are multiple "weights" set for any of the standard mixes (AIL, ELE, RUD) then the self-check process should warn if there is a "Low-rate" setting active

There should be appropriate "countdowns" during each step of the process

The detected stick range values should be displayed so the user can instantly tell if a stick was not moved to the full range.

Values set by Self-check should be visible in the Lua/Device config

**Comment by Richrfl** [comment]
I like this proposal. Way more and better than what I was asking for. Thank you Ben

**Comment by frskyrc-ben** [comment]
keep alive. I would still like to see some better visual confirmation during the 6axis calibration and stick range calibration to know what values are detected. It can really help diagnose incorrect calibration process.
